planning/behavior_path_planner/include/behavior_path_planner/behavior_path_planner_node.hpp

```diff
@@ -49,6 +49,7 @@
 #include <lanelet2_traffic_rules/TrafficRulesFactory.h>
```

Comment:
`planner_data_` is still used in the following functions.
Mutex is required, or we can give a local variable `planner_data` to functions.
getPath
getPathCandidate
publishModuleStatus
updateCurrentPose

Reply:
[b6cd6b8](https://github.com/autowarefoundation/autoware.universe/pull/657/commits/b6cd6b8646ad7708f413d9ce680b39605e2fd689)